packages/firestore/src/api/user_data_reader.ts

```diff
@@ -803,10 +813,16 @@ function createError(
   reason: string,
   methodName: string,
   path?: FieldPath,
-  targetDoc?: DocumentKey
+  targetDoc?: DocumentKey,
+  hasConverter = false
```

Comment:
Would prefer is this was not optional and the third argument. It is easy to miss setting otherwise.

Reply:
done.